backend.native/tests/build.gradle

```diff
@@ -4390,6 +4390,30 @@ if (isAppleTarget(project)) {
         swiftSources = ['framework/multiple']
     }
 
+    frameworkTest("testMultipleFrameworksStatic") {
+        if (cacheTesting != null) {
+            // See https://youtrack.jetbrains.com/issue/KT-34261.
+            expectedExitStatus = 134
+        }
+
+        framework('FirstStatic') {
+            artifact = 'First'
+            sources = ['framework/multiple/framework1', 'framework/multiple/shared']
+            bitcode = true
+            isStatic = true
+            opts = ['-Xstatic-framework', "-Xpre-link-caches=enable"]
+        }
+        framework('SecondStatic') {
+            artifact = 'Second'
+            sources = ['framework/multiple/framework2', 'framework/multiple/shared']
+            bitcode = true
+            isStatic = true
+            opts = ['-Xstatic-framework', "-Xpre-link-caches=enable"]
```

Comment:
Why do you pass `isStatic = true` together with `-Xstatic-framework`?

Reply:
Because `isStatic` doesn't actually enable passing this flag. See also `testObjCExportStatic`.